app/controllers/favorites_controller.rb

```diff
@@ -13,6 +13,13 @@ def create
     redirect_to tweets_path
   end
 
+  def destroy
+    tweet = Tweet.find_by(number: params[:id])
```

Comment:
En general en este caso se utiliza `find_by!` https://apidock.com/rails/v4.0.2/ActiveRecord/FinderMethods/find_by!

Si alguien te pasa un number que no existe, lo esperado es que la página responda con un HTTP 404. 

Si Rails escucha una exception `ActiveRecord::RecordNotFound`, traduce eso en un HTTP 404, que quiere decir Resource not found según REST.